scrapy/utils/deprecate.py

```diff
@@ -126,9 +126,7 @@ def _clspath(cls, forced=None):
     return f'{cls.__module__}.{cls.__name__}'
 
 
-DEPRECATION_RULES = [
-    ('scrapy.telnet.', 'scrapy.extensions.telnet.'),
-]
+DEPRECATION_RULES = []
```

Comment:
Based on the failing `typing` job, we need to mark this as `List[Tuple[str, str]]` now that the type cannot be inferred from the value.

Reply:
@Gallaecio I have now added the type annotations to the PR.